website/app/styles/typography/mixins.scss

```diff
@@ -170,7 +170,7 @@
 }
 
 @mixin doc-font-style-label {
-  @include doc-font-family("gilmer");
+  @include doc-font-family("hashicorp-sans");
```

Comment:
Because this font was meant for Display text, it's a little hard to read at this size (it can be seen on the [icon library](https://hds-website-git-rebranding-followup-hashicorp.vercel.app/icons/library) page). Can this be updated to use the `sans` font-family instead?

Reply:
done